src/main/java/mods/railcraft/common/plugins/jei/blastfurnace/rockcrusher/BlastFurnaceMachineCategory.java

```diff
@@ -0,0 +1,80 @@
+/*------------------------------------------------------------------------------
+ Copyright (c) CovertJaguar, 2011-2017
+ http://railcraft.info
+
+ This code is the property of CovertJaguar
+ and may only be used with explicit written
+ permission unless otherwise specified on the
+ license page at http://railcraft.info/wiki/info:license.
+ -----------------------------------------------------------------------------*/
+
+package mods.railcraft.common.plugins.jei.blastfurnace.rockcrusher;
```

Comment:
Package looks weird, but it can always be fixed.

Reply:
Oh weird, that shouldn't have happened